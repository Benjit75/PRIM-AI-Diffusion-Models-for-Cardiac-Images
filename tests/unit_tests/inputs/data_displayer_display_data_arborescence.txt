{'data_name': 'data_loader_loaded.data', 'start_level': 0, 'start_prefix': ""}