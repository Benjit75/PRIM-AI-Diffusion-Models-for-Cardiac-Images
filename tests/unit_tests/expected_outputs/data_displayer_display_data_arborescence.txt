data_loader_loaded.data
├── train
│	├── patient001
│	│	├── image_data
│	│	│	├── ED
│	│	│	├── ED_gt
│	│	│	├── ES
│	│	│	├── ES_gt
│	│	├── height
│	│	├── weight
│	│	├── group
│	│	├── nb_frames
│	├── patient002
│	│	├── image_data
│	│	│	├── ED
│	│	│	├── ED_gt
│	│	│	├── ES
│	│	│	├── ES_gt
│	│	├── height
│	│	├── weight
│	│	├── group
│	│	├── nb_frames
├── test
│	├── patient101
│	│	├── image_data
│	│	│	├── ED
│	│	│	├── ED_gt
│	│	│	├── ES
│	│	│	├── ES_gt
│	│	├── height
│	│	├── weight
│	│	├── group
│	│	├── nb_frames
│	├── patient102
│	│	├── image_data
│	│	│	├── ED
│	│	│	├── ED_gt
│	│	│	├── ES
│	│	│	├── ES_gt
│	│	├── height
│	│	├── weight
│	│	├── group
│	│	├── nb_frames